### PR TITLE
fix(@desktop/warnings): Fix bunc of qml/nim warnings

### PR DESCRIPTION
**Pull request by borismelnik**
Closes: #5162

### What does the PR do

Fix a bunch of warnings

### Affected areas

QML

### CODE REVIEW

File: ui/app/AppLayouts/Chat/views/ChatView.qml [hunk 1]
```diff
@@ -46,6 +46,7 @@ StatusAppThreePanelLayout {
 
     Connections {
         target: root.rootStore.chatCommunitySectionModule
+        ignoreUnknownSignals: true
```
Comment (osmaczko) [inline ui/app/AppLayouts/Chat/views/ChatView.qml:49]: Do we expect `chatCommunitySectionModule` object to be a different type depending on scenario? If not, then it's not the correct usage (just masking the real issue), see: https://doc.qt.io/qt-5/qml-qtqml-connections.html#ignoreUnknownSignals-prop

What is the root cause of the warning here?

**Review by osmaczko — COMMENTED** [review]
Thanks for fixing these annoying warnings.

**Review by stefandunca — APPROVED** [review]

**Review by endulab — APPROVED** [review]
@borismelnik good job !

File: ui/app/AppLayouts/Chat/views/ChatView.qml (hunk 1 above)
Comment (borismelnik) [reply ui/app/AppLayouts/Chat/views/ChatView.qml:49]: @osmaczko I did it cos `chatCommunitySectionModule` can be `null`.

Comment (borismelnik) [reply ui/app/AppLayouts/Chat/views/ChatView.qml:49]: I think I can change it to `enabled: !!chatCommunityModule`

Comment (osmaczko) [reply ui/app/AppLayouts/Chat/views/ChatView.qml:49]: Then, I believe, `enabled` property should be used instead.
`enabled: root.rootStore.chatCommunitySectionModule != null`

Also according to `target` property docs, it should not produce any warning in that case, but I can get it wrong:
> If set to null, no connection is made and any signal handlers are ignored until the target is not null.

_Please remove `ignoreUnknownSignals` from different places in this PR as well_

Comment (osmaczko) [reply ui/app/AppLayouts/Chat/views/ChatView.qml:49]: @iurimatias this comment should be resolved (I should have used "require changes"). @borismelnik could you please create a followup PR?

**Pull request merged**